debian/nnstreamer-dev.install

```diff
@@ -9,6 +9,8 @@
 /usr/include/nnstreamer/tensor_decoder_custom.h
 /usr/include/nnstreamer/tensor_filter_cpp.hh
 /usr/include/nnstreamer/nnstreamer_cppplugin_api_filter.hh
+/usr/include/nnstreamer/nnstreamer_edge.h
+/usr/include/nnstreamer/nnstreamer_edge_common.h
```

Comment:
Do not export edge_common.h. If necessary, move edge-event functions into nnstreamer_edge.h.

Reply:
Right. Removed.